src/library_dylink.js

```diff
@@ -92,6 +92,10 @@ var LibraryDylink = {
         if (e !== e+0) throw e;
 #endif
         _setThrew(1, 0);
+        // In theory this if could be done on creating the function, 
+        // but I just added this to save wasting code space
+        // and it only happens on an exception
+        if(sig[0]=="j")return BigInt(0);
```

Comment:
Can you wrap this new code in `#if WASM_BIGINT`?   Also please add a space after the `if` and before the `return`.

Also, you can just write `0n` instead of spelling out `BigInt` I believe.

Reply:
Ok, done